handler/cantabular_metadata_export.go

```diff
@@ -143,13 +147,31 @@ func (h *CantabularMetadataExport) Handle(ctx context.Context, workerID int, msg
 				}
 			}
 		}
+
+		areaTypeFound = false
+
 		for _, d := range m.Version.Dimensions {
 			if *d.IsAreaType {
 				d.Label = areaTypeLabel
 				d.Description = areaTypeDescription
+				d.ID = areaTypeId
```

Comment:
Are you 100% sure this is working as expected? I'd half expect this not to update the dimensions of the original struct you're trying to, I might be wrong though.